palantirnet/st-bernards
# MSSB-45: Fix Link Titles

## JIRA Ticket(s)

- See ticket https://palantir.atlassian.net/browse/MSSB-45

## Description

Fix link titles on views and clean up templates.

## Testing instructions

1. Run `composer install`
1. Run `drush cr && drush cim -y && drush cr`
1. Visit: http://st-bernards.local/providers, http://st-bernards.local/locations, http://st-bernards.local/services and verify the links have sensible descriptions for tooltips.
